server/api/src/rsCollCreate.cpp

```diff
@@ -147,6 +147,10 @@ rsCollCreate( rsComm_t *rsComm, collInp_t *collCreateInp ) {
 
 int
 l3Mkdir( rsComm_t *rsComm, dataObjInfo_t *dataObjInfo ) {
+    if ( NULL == dataObjInfo ) {
+        return -1;
```

Comment:
is there a better value to return than -1?  @jasoncoposky @tempoz

Reply:
use SYS_NULL_INPUT instead.